snomed-ct-integration/src/test/java/dev/ikm/tinkar/snomedct/integration/DescriptionSemanticIT.java

```diff
@@ -38,6 +54,62 @@ public static void setup() {
     public static void shutdown() {
         PrimitiveData.stop();
     }
+    @Test
+    public void testDescriptionSemantics()throws IOException {
+        String sourceFilePath ="C:\\Users\\patrichards\\Solor\\SnomedCT_InternationalRF2_PRODUCTION_20240201T120000Z\\Full\\Terminology\\sct2_Description_Full-en_INT_20240201.txt";
```

Comment:
This will be a good end state, but for now let's just iterate through the small sample file attached to the Jira ticket. You can check that file into git under src/test/resources.

Reply:
You still need to update this String to the sample file.